polaris-migrator/src/migrations/v11-styles-replace-custom-property-color/v11-styles-replace-custom-property-color.ts

```diff
@@ -159,32 +127,72 @@ const borderColorMap = {
   '--p-border-highlight-subdued': '--p-color-border-info-subdued',
   '--p-border-success': '--p-color-border-success',
   '--p-border-success-subdued': '--p-color-border-success-subdued',
-};
 
-const fillColorMap = {
-  ...allMap,
+  // Fill
   '--p-icon': '--p-color-icon',
   '--p-icon-on-dark': '--p-color-icon-inverse',
   '--p-icon-hovered': '--p-color-icon-hover',
   '--p-icon-disabled': '--p-color-icon-disabled',
   '--p-icon-subdued': '--p-color-icon-subdued',
-  '--p-interactive': '--p-color-icon-interactive',
-  '--p-interactive-disabled': '--p-color-icon-interactive-disabled',
-  '--p-interactive-hovered': '--p-color-icon-interactive-hover',
-  '--p-interactive-pressed': '--p-color-icon-interactive-active',
   '--p-icon-on-interactive': '--p-color-icon-on-color',
-  '--p-action-primary': '--p-color-icon-primary',
   '--p-icon-on-primary': '--p-color-icon-on-color',
   '--p-icon-critical': '--p-color-icon-critical',
   '--p-icon-on-critical': '--p-color-icon-on-color',
-  '--p-interactive-critical': '--p-color-icon-critical',
   '--p-icon-warning': '--p-color-icon-caution',
   '--p-icon-highlight': '--p-color-icon-info',
   '--p-icon-success': '--p-color-icon-success',
   '--p-icon-attention': '--p-color-icon-caution',
   '--p-icon-pressed': '--p-color-icon-active',
 };
 
+const colorMap = {
+  ...allMap,
+  '--p-interactive': '--p-color-text-interactive',
+  '--p-interactive-on-dark': '--p-color-text-interactive-inverse',
+  '--p-interactive-disabled': '--p-color-text-interactive-disabled',
+  '--p-interactive-hovered': '--p-color-text-interactive-hover',
+  '--p-interactive-pressed': '--p-color-text-interactive-active',
+  '--p-action-primary': '--p-color-text-primary',
+  '--p-interactive-critical': '--p-color-text-critical',
+  '--p-interactive-critical-disabled': '--p-color-text-disabled',
+  '--p-interactive-critical-pressed': '--p-color-text-critical-active',
+  '--p-interactive-pressed-on-dark': '--p-color-text-interactive-inverse',
+};
+
+const backgroundColorMap = {
+  ...allMap,
+  '--p-interactive': '--p-color-bg-interactive',
+  '--p-interactive-disabled': '--p-color-bg-interactive-disabled',
+  '--p-interactive-hovered': '--p-color-bg-interactive-hover',
+  '--p-interactive-pressed': '--p-color-bg-interactive-active',
+  '--p-action-primary': '--p-color-bg-primary',
+  '--p-action-primary-disabled': '--p-color-bg-disabled',
+  '--p-action-primary-hovered': '--p-color-bg-primary-hover',
+  '--p-action-primary-pressed': '--p-color-bg-primary-active',
+  '--p-action-primary-depressed': '--p-color-bg-primary-active',
+  '--p-interactive-critical': '--p-color-bg-critical',
+  '--p-interactive-critical-hovered': '--p-color-bg-critical-strong-hover',
+};
+
+const borderColorMap = {
+  ...allMap,
```

Comment:
I thought it would make more sense to not add here but have another object. Then it could be merged when needed by the user and not merged too early?